specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-02-15-preview/share.json

```diff
@@ -0,0 +1,2019 @@
+          },
+          "ReceivedShares_GetAdlsGen2Account": {
+            "$ref": "./examples/ReceivedShares_GetAdlsGen2Account.json"
+          },
+          "ReceivedShares_GetBlobAccount": {
+            "$ref": "./examples/ReceivedShares_GetBlobAccount.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ReceivedShare"
+        ],
+        "summary": "Create or replace a received share.",
+        "description": "Update changes to a received share",
+        "operationId": "ReceivedShares_Create",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/receivedShareId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "receivedShare",
+            "description": "The received share to create or replace",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReceivedShare"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ReceivedShare"
+            },
+            "headers": {
+              "Operation-Location": {
+                "description": "The URL to track the status of the long running operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ReceivedShares_Create": {
+            "$ref": "./examples/ReceivedShares_Create.json"
+          },
+          "ReceivedShares_UpdateAdlsGen2Account": {
+            "$ref": "./examples/ReceivedShares_UpdateAdlsGen2Account.json"
+          },
+          "ReceivedShares_UpdateBlobAccount": {
+            "$ref": "./examples/ReceivedShares_UpdateBlobAccount.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "ReceivedShare"
+        ],
+        "summary": "Deletes a received share",
+        "description": "Delete a received share",
+        "operationId": "ReceivedShares_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/receivedShareId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
```

Comment:
A 202 response should include a schema for the operation status monitor.

Reply:
@mikekistler When this delete call is made the body returned is empty. Do we still need this? We are adding a header Operation-Location that contains the URL for the status monitor.

Reply:
A "long-running" delete should return a 202 with a representation of the "status monitor" for the operation in the response body. When the operation completes, its result would go in the `result` field of the status monitor, but if there is no response -- as in the case of delete -- then `result` can be omitted.